src/language-handlebars/printer-glimmer.js

```diff
@@ -148,10 +148,10 @@ function print(path, options, print) {
       const isConcat = pp && pp.type === "ConcatStatement";
       return group(
         concat([
-          /*n.escaped ? "{{{" : */ "{{",
+          n.escaped === false ? "{{{" : "{{",
           printPathParams(path, print),
           isConcat ? "" : softline,
-          /*.escaped ? "}}}" :*/ "}}"
+          n.escaped === false ? "}}}" : "}}"
```

Comment:
This could just be `n.escaped ? "}}" : "}}}"`.

Reply:
That's what I tried initially, but it resulted in test failures because prettier would output `{{{` where it should have output `{{`. My hunch is that `escaped` is only set when it's `false`, otherwise it's `undefined` or `null`.